examples/wasm/index.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
```

Comment:
If you don't have a strong policy, how about using google style?
(I want to use tab=2spaces)

https://google.github.io/styleguide/htmlcssguide.html

Reply:
done